client/state/sites/updates/test/actions.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_UPDATES_RECEIVE,
+	SITE_UPDATES_REQUEST,
+	SITE_UPDATES_REQUEST_SUCCESS,
+	SITE_UPDATES_REQUEST_FAILURE,
+} from 'state/action-types';
+import {
+	siteUpdatesReceiveAction,
+	siteUpdatesRequestAction,
+	siteUpdatesRequestSuccessAction,
+	siteUpdatesRequestFailureAction,
+} from '../actions';
+
+describe( 'actions', () => {
+	const siteId = 2916284;
+
+	describe( '#siteUpdatesReceiveAction()', () => {
+		it( 'should return a site updates receive action object', () => {
+			const updates = {
+				plugins: 1,
+			};
+			const action = siteUpdatesReceiveAction( siteId, updates );
+
+			expect( action ).to.eql( {
```

Comment:
Good call, will consider using that the next time. Not worth updating it right now, since they're both checking deep equality.